Pd 391 user feedback tutorial (#2014)
truenas/documentation: content/_includes/FeedbackWidget.md

---
---

{{< trueimage src="/images/SCALE/23.10/FeedbackWidget.png" alt="The Feedback Widget" id="2: The Feedback Widget" >}}

> Use this widget to submit a star rating and comments or suggestions for the current page.

Reviewer: ```suggestion
Use this window to submit a star rating and comments or suggestions for the current page.
```

{{< hint type=note >}}

> The feedback widget is used for general user experience feedback. Submit bug tickets through the **File Ticket** form. The **File Ticket** link opens the **File Ticket** form that you can also access from the **System Settings > General** screen **File Ticket** button. See [Filing an Issue Ticket in SCALE]({{< relref "fileissuescale.md" >}}) for further instructions.

Reviewer: Maybe tighten this up a bit. How about this
```suggestion
The feedback window is used for general user experience feedback. 
Submit bug tickets through the **File Ticket** link that opens the **File Ticket** form, also accessed from the **System Settings > General** screen **File Ticket** button. See [Filing an Issue Ticket in SCALE]({{< relref "fileissuescale.md" >}}) for further instructions.
```
